nss-day-cohort-43-csharp/Fullstack-Panteras-Implicit-Return
# Se auth

# Description
Users can register and login. Basic auth.
## Type of change
Please delete options that are not relevant.
# Testing Instructions
1. Register as a new user
2. Check firebase to ensure it was created & database
3. login as new user
4. try to go to other URL links "/explore" as a non-logged in user
# Checklist:
